app/src/main/java/org/wikipedia/main/MainFragment.java

```diff
@@ -144,13 +146,29 @@ public static MainFragment newInstance() {
             viewPager.setCurrentItem(item.getOrder(), false);
             return true;
         });
+        setBottomNavBackground();
 
         if (savedInstanceState == null) {
             handleIntent(requireActivity().getIntent());
         }
         return view;
     }
 
+    /**
+     * Minimize GPU over draw for Bottom Navigation Bar
+     */
+    private void setBottomNavBackground() {
+        int[] attrs = {R.attr.nav_tab_background_color};
+        TypedArray ta = requireContext().obtainStyledAttributes(attrs);
+        int transparent = ContextCompat.getColor(requireContext(), android.R.color.transparent);
+        int taColor = ta.getColor(0, transparent);
+        ta.recycle();
+        navTabContainer.setBackgroundColor(taColor);
```

Comment:
I'm not seeing why it's necessary to set this color here, instead of in XML which is much more simple.

Reply:
Tried `android:background="@android:color/transparent"` in the XML but it had no effect on overdraw here

Reply:
Turn out I needed `app:elevation="0dp"` to get rid of the inner shadow background
![weird background of BottomNavigationView](https://user-images.githubusercontent.com/19919208/88247127-a3819680-cc6a-11ea-96b2-dfb18d8d46dc.png)